### PR TITLE
#186 Mediaserver for Cooper

### PR DESCRIPTION
**Pull request by emptycoder**

### CODE REVIEW

File: Cooper/ClientApp/src.v2/app/components/upload-form/upload-form.component.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  selector: 'coop-upload-form',
+  templateUrl: './upload-form.component.html',
+  styleUrls: ['./upload-form.component.scss']
+})
+export class UploadComponent {
+  public error: string;
+
+  constructor(private dialogRef: MatDialogRef<UploadComponent>,
+              @Inject(MAT_DIALOG_DATA) private data: string,
+              private mediaserver: MediaserverService) { }
```
Comment (horba) [inline Cooper/ClientApp/src.v2/app/components/upload-form/upload-form.component.ts]: no business logic in dummy service, all interactions via @Input() and @Output()

File: Cooper.Services.Interfaces/Cooper.Services.Interfaces.csproj [hunk 2]
```diff
@@ -12,4 +12,10 @@
     <ProjectReference Include="..\Cooper.Models\Cooper.Models.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore.Http.Features">
+      <HintPath>..\..\..\..\..\Program Files\dotnet\sdk\NuGetFallbackFolder\microsoft.aspnetcore.http.features\2.2.0\lib\netstandard2.0\Microsoft.AspNetCore.Http.Features.dll</HintPath>
```
Comment (Hunroll) [inline Cooper.Services.Interfaces/Cooper.Services.Interfaces.csproj]: Nuget: 
image

**Review by Hunroll — CHANGES_REQUESTED** [review]

File: Cooper/ClientApp/src.v2/app/components/upload-form/upload-form.component.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'coop-upload-form',
+  templateUrl: './upload-form.component.html',
+  styleUrls: ['./upload-form.component.scss']
+})
+export class UploadComponent {
+  @Input() public error: string;
```
Comment (horba) [inline Cooper/ClientApp/src.v2/app/components/upload-form/upload-form.component.ts]: error-message

File: Cooper/ClientApp/src.v2/app/components/upload-form/upload-form.component.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  @Output() public upload: EventEmitter<any> = new EventEmitter();
```
Comment (horba) [inline Cooper/ClientApp/src.v2/app/components/upload-form/upload-form.component.ts]: let's decide with team about common naming of custom events :)

**Review by Hunroll — APPROVED** [review]

File: Cooper/ClientApp/src.v2/app/components/upload-form/upload-form.component.scss [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      color: #fff;
+      margin-left: calc(50% - 110px);
+      border: none;
+      padding: 5px;
+      padding-right: 15px;
+      padding-left: 15px;
+    }
+
+    input:hover {
+      background: #48494c
```
Comment (horba) [inline Cooper/ClientApp/src.v2/app/components/upload-form/upload-form.component.scss:30]: ;

**Review by horba — APPROVED** [review]
after ; fix can be merges

**Review by horba — APPROVED** [review]

**Pull request merged**